enterprise-suite/scripts/lbc.py

```diff
@@ -360,7 +348,21 @@ def install(creds_file):
         else:
             fail('Unable to install/upgrade console, an install named {} with status {} already exists. '
                  .format(args.helm_name, status))
-    
+
+        if args.wait:
+            helm_args += ' --wait'
+
+        if args.reuse_resources or status == 'failed':
+            # Reuse PVCs if present
+            if are_pvcs_created(args.namespace):
+                printerr('warning: found existing PVCs from previous console installation, will reuse them')
```

Comment:
I'm not sure it should be a warning - since it might be okay and expected behaviour.

Reply:
Changed to `info:`